package.json

```diff
@@ -20,10 +20,10 @@
     "prepare": "husky"
   },
   "author": "",
-  "license": "ISC",
+  "license": "Apache-2.0",
   "dependencies": {
-    "@tazama-lf/frms-coe-lib": "5.0.0",
-    "@tazama-lf/frms-coe-startup-lib": "2.4.0-rc.2",
+    "@tazama-lf/frms-coe-lib": "^5.1.0-rc.1",
+    "@tazama-lf/frms-coe-startup-lib": "^2.4.0-rc.2",
```

Comment:
Please remove the caret for the libs. We want to match on the exact library version